src/AasxServerBlazor/Pages/Db.razor

```diff
@@ -1,905 +1,803 @@
+                    </span>
+                    <br />
+                    <br />
+                    @if (size > 1000 && data.Count > 1000)
+                    {
+                        <b><span style="color:red;">First 1000 as TABLE</span></b>
+                        <br />
+                    }
+                    <table class="table table-bordered table-sm" style="width:100%;word-wrap:break-word;word-break:break-all;">
+                        <thead>
+                            <tr>
+                                <th style="word-break:keep-all">SM#</th>
+                                <th style="word-break:keep-all">Par#</th>
+                                <th style="word-break:keep-all">SME#</th>
+                                <th style="word-break:keep-all">Type</th>
+                                <th style="word-break:keep-all">IdShort</th>
+                                <th style="word-break:keep-all">SemanticId</th>
+                                <th style="word-break:keep-all">Value</th>
+                                <th style="word-break:keep-all">ValueType</th>
+                                <th style="word-break:keep-all">Links</th>
+                                <th style="word-break:keep-all">API</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            @for (int i = 0; i < data.Count && i < 1000; i++)
+                            {
+                                var smeDB = data[i];
+                                string path = smeDB.IdShort;
+                                int? pnum = smeDB.ParentSMEId;
+                                while (pnum != null)
+                                {
+                                    var smeDBP = db.SMESets.Where(s => s.Id == pnum).First();
+                                    path = smeDBP.IdShort + "." + path;
+                                    pnum = smeDBP.ParentSMEId;
+                                }
+                                var smDB = db.SMSets.Where(s => s.Id == smeDB.SMId).First();
+                                string submodel64 = Base64UrlEncoder.Encode((smDB.Identifier));
+                                string link = AasxServer.Program.externalBlazor + "/submodels/" + submodel64 + "/submodel-elements/" + path;
+                                <tr>
+                                    <td style="word-break:keep-all">@smeDB.SMId</td>
+                                    <td style="word-break:keep-all">@smeDB.ParentSMEId</td>
+                                    <td style="word-break:keep-all"><b>@smeDB.Id</b></td>
+                                    <td style="word-break:keep-all">@smeDB.SMEType</td>
+                                    <td>@smeDB.IdShort</td>
+                                    <td>@smeDB.SemanticId</td>
+                                    <td>
+                                        @if (smeDB.SMEType == "MLP")

```

Comment:
**must**: Please use `Equals` for string comparison.
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca2251